woocommerce/google-listings-and-ads
# Coupon notification & channel visibility bulk update

### Changes proposed in this Pull Request:

1. add note for coupon upload feature
2. add bulk edit option for coupon channel visibility 


### Screenshots:


![image](https://user-images.githubusercontent.com/87394744/142817488-ea68ff2b-8fa7-4b42-8ca7-0cee962ab345.png)
![image](https://user-images.githubusercontent.com/87394744/142817535-dd5b022b-7703-49dd-80f6-b0ce62281cb0.png)


### Detailed test instructions:
1. Go to coupons page
2. Select several coupons, including a coupon that does not shown on Google
3. Select "bulk actions"->"edit", click apply
4. Default Google visibility equals to "no-change"
5. Select visibility  = show on Google and apply the change
6. verify selected coupon has visibility updated and coupon is sent to Google or pending for sync.



### Changelog entry

> Add - Bulk update channel visibility for coupons.
